shared/teams/join-team/index.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+import React from 'react'
+import {Box, Button, HeaderHoc, Icon, Input, PopupDialog, Text, ScrollView} from '../../common-adapters/index'
+import {isMobile} from '../../constants/platform'
+import {globalColors, globalMargins, globalStyles} from '../../styles'
+
+import type {Props} from './'
+
+const styleContainer = {
+  ...globalStyles.flexBoxCenter,
+  ...(isMobile ? {} : {cursor: 'default'}),
```

Comment:
why is this needed?

Reply:
it's not, I don't think. The basic styling was based off of the new-team dialog, which [includes this](https://github.com/keybase/client/blob/master/shared/teams/new-team/index.js#L59). I wasn't sure what was safe to remove, but it looks like it's dead code here.